ansible/roles/ocp4-workload-debezium-demo/tasks/provision_registry.yaml

```diff
@@ -36,18 +37,13 @@
     resource_definition: "{{ lookup('template', 'kafka-demo.yaml.j2') }}"
 
 - name: Wait for Kafka Cluster to start
-  k8s_facts:
+  k8s_info:
     kind: Kafka
     name: demo
     namespace: '{{ ocp4_workload_debezium_demo_registry_namespace }}'
     api_version: kafka.strimzi.io/v1beta1
-  register: kafka_resource
-  until: >-
-    (kafka_resource.resources is defined) 
-    and (kafka_resource.resources|length>0)
-    and (kafka_resource.resources[0].status is defined) 
-    and (kafka_resource.resources[0].status.conditions is defined)
-    and (kafka_resource.resources[0].status.conditions[0].type == 'Ready') 
+  register: k8s_info
```

Comment:
If these vars are going to be renamed, can we be consistent? In the earlier task you are using `r_k8s_info` and here `k8s_info`. I think if you register these, they should be a little more specific to what is being checked like `r_crd_apicurio` or something in this case.

Reply:
+1

Reply:
Fixed to use the `r_k8s_info` consistently. I think using a generic variable name is appropriate here because it highlights that this registered variable is not used for anything else beyond this task. It helps underscore that it is just used here and done with it.